### DISCUSSION TITLE
Imported module directory created on every run

### DISCUSSION THREAD

**Discussion by nazikus** [pyinstaller/pyinstaller#7117] [Help] [answered]
After adding binary module [`jpegtran-cffi`](https://github.com/jbaiter/jpegtran-cffi), every time I run pyinstaller app it creates directory `jpegtran` in current directory. Related to #7025.

Directory content:
```bash
~/dev/jpegtran/
└── __pycache__
    └── _cffi__x30e896f0xf3ca0677.c
```

Any ideas how to suppress this file appearing all the time?]

**Comment by rokm** [comment]
Can you provide a minimal example that reproduces the problem? A basic `import jpegtran` program does not seem to create a directory like that...

**Comment by nazikus** [comment, marked as answer]
Solved by @rokm here https://github.com/pyinstaller/pyinstaller/issues/7025#issuecomment-1268285674. It was a library-specific issue.
